### PR TITLE
fix: typo spelling grammar

### PR DESCRIPTION
**Pull request by slowy07**
## description 
hello !
fix typo spelling grammar, and replace to correct words

## reason
for more readable code comments and documentation

### CODE REVIEW

File: net/flowtrack/flowtrack.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ type entry struct {
 	value interface{}
 }
 
-// Add adds a value to the cache, set or updating its assoicated
+// Add adds a value to the cache, set or updating its assoiciated
```
Comment (DentonGentry) [inline net/flowtrack/flowtrack.go]: "associated", I think

Comment (slowy07) [reply net/flowtrack/flowtrack.go]: okay sir @DentonGentry

Comment (slowy07) [reply net/flowtrack/flowtrack.go]: okay sir @DentonGentry ,i will do it 👍🏻

**Comment by slowy07** [comment]
changes on [9a8934b](https://github.com/tailscale/tailscale/pull/2701/commits/9a8934b69e254217cc9ebfabbb5eff5b67389823)

**Comment by DentonGentry** [comment]
We do need a Developer Certificate of Origin, which is the Signed-off-by line you see in commit messages in this repository: https://probot.github.io/apps/dco/

You can add one using: `git commit --amend --signoff` and force-pushing.

**Comment by bradfitz** [comment]
Can you also squash it down to one commit? It shouldn't contain a partial revert in the series.

**Comment by slowy07** [comment]
sorry sir @DentonGentry , i do ``git commit --amend --signoff`` and ``push --force-with-lease``,get didn't work

**Comment by DentonGentry** [comment]
There are two commits in this PR, and the first does not have a Signed-off-by.

If you squash this to one commit, and ensure that one commit has a Signed-off-by, it should pass.

**Comment by slowy07** [comment]
thanks sir!, how i can do it ??

**Comment by DentonGentry** [comment]
Make sure that `git status` shows no other files modified. If there are modified files, use `git stash` to store them temporarily.

```
git reset --soft HEAD^1
git add .
git commit --amend --signoff
git push -f
```

This is what I think will be simpler. The usual way to do this is interactive rebase: https://stackoverflow.com/questions/2563632/how-can-i-merge-two-commits-into-one-if-i-already-started-rebase

**Comment by slowy07** [comment]
thanks sir !, it's okay the commit ``--amend --signoff`` by ``Brad Fitzpark <bardflitz@tailscale.com> ?? 

thanks !

**Comment by DentonGentry** [comment]
The Signed-off-by needs to be you. You're asserting that you wrote the code, and that you are submitting code to which you own the copyright.

An obviously fake signoff like "Brad Fitzpark <bardflitz@tailscale.com>" isn't suitable.

**Comment by slowy07** [comment]
sorry @DentonGentry , can i closed the pull ?, because it's not changes, and still fail

**Pull request closed without merging**